recipe/meta.yaml

```diff
@@ -10,18 +10,20 @@ source:
   sha256: f271b298b97f5955d53fb12b72c1fb1948c22c1a6b70b315c54cedaca0264ef5
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [py<38]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps
   number: 0
 
 requirements:
   host:
-    - python >=3.7
+    - python 
     - pip
-    - flit-core >=3.2, <4
+    - setuptools
+    - wheel
+    - flit-core
```

Comment:
so it does not have pinning because its python dependency, right ?

Reply:
I think we actually want to have a more exact pinning in the host section (according to the linter)